packages/shared/components/popups/AddNodePopup.svelte

```diff
@@ -57,25 +58,19 @@
         isDeveloperProfile={$activeProfile.isDeveloperProfile}
     />
     <div class="flex flex-row justify-between space-x-4 w-full">
-        <Button secondary classes="w-1/2" onClick={closePopup} disabled={isBusy}>
+        <Button outline classes="w-1/2" onClick={closePopup} disabled={isBusy}>
             {localize('actions.cancel')}
         </Button>
         <Button
             disabled={!node.url || isBusy}
-            type="submit"
+            type={HTMLButtonType.Submit}
             form="node-configuration-form"
             classes="w-1/2"
             onClick={handleAddNode}
+            {isBusy}
```

Comment:
In general the settings action buttons were a bit more narrow vertically to ease a but the views. 
![image](https://user-images.githubusercontent.com/3624944/190414751-4a71ccf2-99d2-404f-93bc-b98249ac4e00.png)
UI kit 
![image](https://user-images.githubusercontent.com/3624944/190415151-f6be07bc-8206-44f2-9c7e-29e20fe447d1.png)

Reply:
This applies to all buttons that appear on the main settings view with the settings navigation on the left eg 
![image](https://user-images.githubusercontent.com/3624944/190415478-b36aa31f-ddb5-4542-a14c-05f15bc237ef.png)